REFERENCE.md

```diff
@@ -184,6 +200,38 @@ Data type: `String`
 
 String of the comment in passwd file of the local qualys user
 
+### <a name="profile_auditqualys_eus_reporting"></a>`profile_audit::qualys_eus_reporting`
+
+Allow qualys to sudo to root and add wrapper for subscription-manager to lie about enabled repos.
```

Comment:
Again description is slightly wrong in that a different option now allows the sudo part.

Reply:
Documentation updated